onedal/datatypes/data_conversion.cpp

```diff
@@ -155,6 +155,7 @@ dal::table convert_to_table(PyObject *obj) {
         if (array_is_behaved(ary) || array_is_behaved_F(ary)) {
 #define MAKE_HOMOGEN_TABLE(CType) res = convert_to_homogen_impl<CType>(ary);
             SET_NPY_FEATURE(PyArray_DESCR(ary)->type,
+                            PyArray_DESCR(ary)->elsize,
```

Comment:
Needed to be checkec with numpy 2.0

Reply:
Good catch Samir - yes please rebase and follow the revisions to using `->` from numpy 2.0 (https://github.com/intel/scikit-learn-intelex/pull/1817)